tests/queries/0_stateless/01321_monotonous_functions_in_order_by.sql

```diff
@@ -0,0 +1,18 @@
+SET enable_debug_queries = 1;
+SET optimize_monotonous_functions_in_order_by = 1;
+
+SELECT number FROM numbers(10) ORDER BY toFloat32(toFloat64(number));
+SELECT number FROM numbers(10) ORDER BY abs(toFloat32(number));
```

Comment:
Here and there. Make less useless lines in fererence: 10 -> 4. It's hard to read reference file.
It's better to make more queries with other cases/functions.

If you know fucntions that should be monotonic but not marked as monotonic yet, make a test for them, comment it with `-- TODO: `